src/index.js

```diff
@@ -143,6 +144,14 @@ const unexpectedEnzyme = {
       }
     });
 
+    childExpect.exportType({
+      name: 'ShallowWrapper',
+      base: 'ReactWrapper',
```

Comment:
Have you checked that the shallow wrapper is a clean subset of the full render?

Reply:
Do you mean for the assertions we support in `ReactWrapper`? Would it be better if we did that manually when adding assertions? `<ReactWrapper|ShallowWrapper>`?

Reply:
We could have one Wrapper base and two types. I think this inheritance will bite us. where you want to special case ShallowWrapper.

Reply:
Sounds sane. I'll fix that later.

Reply:
Probably better to call the base EnzymeWrapper